README.md

```diff
@@ -115,6 +115,39 @@ Server
 In many rails projects when you run the server with `rails s` Rails sets RAILS_ENV to "development".  If you do that with this app, you will connect to the development database on AWS (if you have permission to decrypt the value).  Instead, run `RAILS_ENV=local rails s` to start the server and `RAILS_ENV=local rails c` to run the console.
 
 
+Multithreading
+========================
+Our multithreading functionality is done through ActiveJob, which employs DelayedJob on the backend.
+ActiveJob gets better after rails 5.2, so keep in mind that there is some functionality that is not perfect, until we can upgrade rails.
+
+Backstory:  Rails runs on a single thread.  Even when you schedule asynchronous operations, s.a. sending a request with HTTParty, that request is executed on the same thread as the rest of the application.  Rails simulates multithreading with creative use of scheduling.
+
+To put something on its own thread, you have to first create the second thread by starting the second worker on the same EC2 server:
+```
+RAILS_ENV=qa bin/delayed_job start
```

Comment:
This is great — I think it'd be great if you had a simple set of instructions for running locally for development.

Reply:
I just replace the env with "local".  There are other ways to run the worker, but they're not more useful, and I'd rather keep the instructions easy.

Reply:
Sorry, I wasn't clear. I meant just have a simple set of instructions to run locally like:

```
$ bundle install
$ RAILS_ENV=local bundle exec rails s
then in a separate window run
$ RAILS_ENV=local bundle exec bin/delayed_job start
```
For example. Not sure if I've got those instructions right, but you get the idea.

Reply:
done